File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/9 - 20 and 21 April 2019/03. Easter Trip.py
destination = input()
dates = input()
nights = int(input())

price = 0

if destination == "France":
    if dates == "21-23":
        price = nights * 30
    elif dates == "24-27":
        price = nights * 35
    elif dates == "28-31":
        price = nights * 40
elif destination == "Italy":
    if dates == "21-23":
        price = nights * 28
    elif dates == "24-27":
        price = nights * 32
    elif dates == "28-31":
        price = nights * 39
elif destination == "Germany":
    if dates == "21-23":
        price = nights * 32
    elif dates == "24-27":
        price = nights * 37
    elif dates == "28-31":
        price = nights * 43

print(f"Easter trip to {destination} : {price:.2f} leva.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/01. Online Exam - 20 and 21 April 2019/03. Easter Trip.js
function easterTrip(params) {
    let country = params.shift();
    let period = params.shift();
    let nights = Number(params.shift());

    let prices = {
        'France': {'21-23': 30, '24-27': 35, '28-31': 40},
        'Italy': {'21-23': 28, '24-27': 32, '28-31': 39},
        'Germany': {'21-23': 32, '24-27': 37, '28-31': 43}
    };

    let price = prices[country][period] * nights;

    console.log(`Easter trip to ${country} : ${price.toFixed(2)} leva.`);
}

easterTrip(['Germany', '24-27', '5']);  // Expected output:
// Easter trip to Germany : 185.00 leva.

easterTrip(['Italy', '21-23', '7']);  // Expected output:
// Easter trip to Italy : 196.00 leva.

easterTrip(['France', '28-31', '8']);  // Expected output:
// Easter trip to France : 320.00 leva.